### PR TITLE
[15.0][MIG] stock_request

### PR DESCRIPTION
**Pull request by BernatPForgeFlow**

### CODE REVIEW

**Review by JoanMForgeFlow — APPROVED** [review]
LGTM!

**Review by ajaniszewska-dev — APPROVED** [review]

**Comment by BernatPForgeFlow** [comment]
@LoisRForgeFlow @JordiBForgeFlow @sebalix @pedrobaeza

File: stock_request/models/stock_request_order.py [hunk 1]
```diff
@@ -323,36 +320,24 @@ def _create_from_product_multiselect(self, products):
...
+                            product_uom_id=product.uom_id.id,
+                            product_uom_qty=1.0,
+                            expected_date=expected,
+                        ),
+                    )
+                    for product in products
+                ],
             )
+        )
         action = self.env.ref("stock_request.stock_request_order_action").read()[0]
```
Comment (sebalix) [inline stock_request/models/stock_request_order.py:341]:
```python
        action = self.env["ir.actions.act_window"]._for_xml_id("stock_request.stock_request_order_action")
```
This fix should have been applied also on 14.0 version.

**Review by sebalix — CHANGES_REQUESTED** [review]
Could you squash the following type of commits to reduce the size of the history?

- `[UPD] Update stock_request.pot` from `oca-travis`
- `Update translation files` from `oca-transbot`
- `[UPD] README.rst` from `OCA-git-bot`

File: stock_request/models/stock_request_order.py (hunk 1 above)
Comment (BernatPForgeFlow) [reply stock_request/models/stock_request_order.py:341]: @sebalix  Okay, I'll squash them now. Do I backport the changes to v14?

**Review by JoanMForgeFlow — APPROVED** [review]
Code seems OK!

**Review by MiquelRForgeFlow — APPROVED** [review]

**Comment by MiquelRForgeFlow** [comment]
But IMO, https://github.com/OCA/stock-logistics-warehouse/pull/1350 looks better.

**Comment by JordiBForgeFlow** [comment]
@BernatPForgeFlow can you fetch from https://github.com/OCA/stock-logistics-warehouse/pull/1350 the parts that might be interesting to fetch? @olveracuenca we can then merge this PR.

**Comment by BernatPForgeFlow** [comment]
> @BernatPForgeFlow can you fetch from #1350 the parts that might be interesting to fetch? @olveracuenca we can then merge this PR.

@JordiBForgeFlow Okay, already fetched some parts.

**Pull request merged**